test/e2e_node/services/kubelet.go

```diff
@@ -87,11 +87,11 @@ func RunKubelet() {
 
 const (
 	// Ports of different e2e services.
-	kubeletReadOnlyPort = "10255"
+	kubeletHealthCheckPort = "10248"
```

Comment:
Why do we have to change this?

Reply:
My mistake.
#101139 was mistakenly merged into this PR.
The reason can be found in #101139.

I will drop this commit here.